charts/acp/values.yaml

```diff
@@ -81,108 +92,265 @@ ingress:
+  web_rotation_keys: false
+
+## Cert-manager configuration
 ##
 certManager:
+  ## Enables the cert-manager integration
+  ##
   enabled: false
+
+  ## The requested ‘duration’ (i.e. lifetime) of the Certificate
+  ##
+  duration: 8760h
+
+  ## How long before the currently issued certificate’s expiry cert-manager should renew the certificate.
+  ##
+  renewBefore: 168h
+
+  ## Options to control private keys used for the Certificate.
+  ##
   privateKey:
-    size: 4096
+    size: 2048
+    algorithm: RSA
+  
+  ## extraNames is a list of DNS subjectAltNames to be set on the Certificate.
+  ##
+  extraNames: []
+
+  ## IssuerRef is a reference to the issuer for this certificate
+  ##
   issuerRef:
+    name: ca-issuer
     kind: ClusterIssuer
 
 ## Disables TLS in ACP
 ##
 tlsDisabled: false
 
+## Import Job configuration
+##
 importJob:
+  ## Enables the import job
+  ##
   enabled: false
-  # path to config files
+  
+  ## Config files path
+  ##
   config:
     - /data/config.yaml
     - /data/extraconfig.yaml
     - /secret/config.yaml
-  # import mode (update, fail, ignore)
+  
+  ## Import mode (update, fail, ignore)
+  ##
   mode: update
-  # input file format: yaml or json
+  
+  ## Input file format: yaml or json
+  ##
   format: yaml
-  # path to the input file
+  
+  ## Path to the input file
+  ##
   input: /import/seed.yaml
-  # extra args for import command
+  
+  ## Extra args for import command
+  ##
   extraArgs: []
-  # the data should match import configuration endpoint request body
-  # https://docs.authorization.cloudentity.com/api/system/#operation/importConfiguration
+  
+  ## The data should match import configuration endpoint request body
+  ## https://docs.authorization.cloudentity.com/api/system/#operation/importConfiguration
+  ##
   data:
     tenants: []
     servers: []
     clients: []
+
+  ## ACP import job resource requests and limits
+  ## Ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
   resources: {}
 
-## Default ACP config file
+## Enable Default ACP config 
 ##
 config:
   create: true
-  # the data should match acp configuration options
-  # https://docs.operations.cloudentity.com/reference/acp-configuration/
+  
+  ## The data should match acp configuration options
+  ## https://docs.operations.cloudentity.com/reference/acp-configuration/
+  ##
   data:
   #  logging:
   #     level: debug
 
-## Additional ACP config file from secret
+## ACP config file from secret
 ##
 secretConfig:
+  ## Enable secret config
+  ##
   create: true
-  # the data should match acp configuration options
-  # https://docs.operations.cloudentity.com/reference/acp-configuration/
+
+  ## Secret annotations
+  ##
+  # annotations: {}
+
+  ## The data should match acp configuration options
+  ## https://docs.operations.cloudentity.com/reference/acp-configuration/
+  ##
   data: {}
   #  system:
   #     secret: mysecret
 
+## Dummy cert and key
+##
 certificate:
-  ## Dummy cert and key
+  ## Certificate:
```

Comment:
In the future, I would remove this example cert and key from values.yaml. 
We can keep them as default values in the templates.

Reply:
https://cloudentity.atlassian.net/browse/DEVOPS-1586